client/get.go

```diff
@@ -82,3 +102,47 @@ func runGet(cmd *Command, args []string) {
 	fatalf("Key version not found.")
 
 }
+
+func retrieveTinkKeyset(keyID string) ([]byte, error) {
+	err := isIDforTinkKeyset(keyID)
+	if err != nil {
+		return nil, err
+	}
+	// get primary and active versions of this knox identifier.
+	// Should we get the versions from Network or cache?
+	primaryAndActiveVersions, err := cli.NetworkGetKey(keyID)
```

Comment:
Should we get the stored secret from Network or Cache?

Reply:
cache is probably fine, cc @kalpeshdharwadkar  @devinlundberg

Reply:
Is this case, should we use function [GetKey](https://github.com/pinterest/knox/blob/5611f5557c5ca3a08fc1f78d49f2a5bb630be06c/client.go#L228) here? We can try to get it from Cache first, if fail, we will try Network.

Reply:
yes, that function will use network if cache fails.

Reply:
Thanks, I have committed a new version to use the func GetKey.

Reply:
You should by default use the cache and fallback to the network. You also might want to support the -n option to force network calls.